### PR TITLE
1714 admin operator external user can see operators user access grid

### PR DESCRIPTION
**Pull request by Sepehr-Sobhani**
[ISSUE 1714](https://github.com/bcgov/cas-registration/issues/1714)

**NOTES:**
- Transferred some components and pages from Reg1 while attempting to improve the functionality. There were a few issues and edge cases that were not considered.
- Added tests for all components, functions, and pages. There were no unit tests for this functionality in Reg1.
- Resolved an issue where if a user declined the access request and then undid that action, the user role dropdown became empty and could cause malfunctions for further actions (approving or declining).
- Excluded DECLINED user operators from `get_operator_users` -> Not related to this PR

**TEST:**
To test, log in as an external admin user (bc-cas-dev) and click on the `Users and Access Requests` link in the administration dashboard.

### CODE REVIEW

File: bc_obps/common/fixtures/dashboard/bciers/external.json
[the remark this answers is not in the document]
Comment (Sepehr-Sobhani) [reply bc_obps/common/fixtures/dashboard/bciers/external.json:1]: Thank you for noticing this issue 🙏 Now I have a better understanding of conditions in dashboard tile fixtures 😃 
Here is just one thought that I wanted to share with you: What if we create a separate fixture for external admins and only return it if the user has the admin role? This way, we can reduce one API call. What do you think?

**Comment by shon-button** [comment]
nit:
Breadcrumbs do not match page title:
image

File: bc_obps/common/fixtures/dashboard/bciers/external.json
[the remark this answers is not in the document]
Comment (shon-button) [reply bc_obps/common/fixtures/dashboard/bciers/external.json:1]: @Sepehr-Sobhani 

Separate fixture for external admins is certainly an option.  Currently, we have an example of this option in `bc_obps/common/fixtures/dashboard/administration/internal_admin.json` and `bc_obps/common/fixtures/dashboard/administration/internal.json`

**Comment by Sepehr-Sobhani** [comment]
> nit: Breadcrumbs do not match page title: image

@shon-button What is the best approach to do this, by `title` query parameter or renaming the page name?

**Review by shon-button — APPROVED** [review]

**Pull request merged**